From: Tom Gundersen <teg@jklm.no>
Date: Sun, 6 Jul 2014 14:12:28 +0200
Subject: machine: don't return uninitialized variable

Repotred by Ronny Chevalier

(cherry picked from commit f14aa1f1b2e4e99ee20393871b5f64f1378ed6c3)
(cherry picked from commit 6e07a1c070376b7c2bcc8d57e8adbc8f6f78e8d5)
---
 src/machine/machine.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/machine/machine.c b/src/machine/machine.c
index 3df9c45..56986c8 100644
--- a/src/machine/machine.c
+++ b/src/machine/machine.c
@@ -342,7 +342,7 @@ static int machine_stop_scope(Machine *m) {
         free(m->scope_job);
         m->scope_job = job;
 
-        return r;
+        return 0;
 }
 
 int machine_stop(Machine *m) {
